src/client/activation/requirementsTxtLinkActivator.ts

```diff
@@ -0,0 +1,26 @@
+import { injectable } from 'inversify';
+import { Hover, languages, TextDocument, Position } from 'vscode';
+import { IExtensionSingleActivationService } from './types';
+
+const PYPI_PROJECT_URL = 'https://pypi.org/project';
+
+export function generatePyPiLink(name: string): string | null {
```

Comment:
```suggestion
export function generatePyPiLink(name: string): string | undefined {
```
That should address the build error